src/Features/CSharp/Portable/ExternalAccess/Pythia/PythiaDeclarationNameRecommender.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.CSharp.Completion.Providers.DeclarationName;
+using Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery;
+using Microsoft.CodeAnalysis.CSharp.ExternalAccess.Pythia.Api;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.CSharp.ExternalAccess.Pythia
+{
+    [Export(typeof(PythiaDeclarationNameRecommender)), Shared]
+    internal sealed class PythiaDeclarationNameRecommender : IDeclarationNameRecommender
```

Comment:
why not just mef import many all the IDeclarationNameRecommender, as opposed to exporting the pythia one directly?

our own recommender and the pythia ones can just be pulled in agnostically.